lib/components/dashboard-modules/SunnyTimeframe.tsx

```diff
@@ -46,23 +40,23 @@ const SunnyTimeframe: FC<{ siteUUID: string }> = ({ siteUUID }) => {
   let sunnyText = '';
 
   if (isRelativeTime) {
-    let difference = millisecondsToHours(
-      forecastData.forecast_values[index].datetime_utc.getTime() -
-        new Date().getTime()
-    );
+    let difference = dayjs(
+      forecastData.forecast_values[index].datetime_utc
+    ).diff(dayjs(), 'hours', true);
+
     let timeUnit = difference === 1 ? 'Hour' : 'Hours';
 
     // If we are under an hour, round to minutes
     if (difference < 1) {
-      difference = hoursToMinutes(difference);
+      difference = dayjs.duration(difference, 'hours').asMinutes();
       timeUnit = difference ? 'Minute' : 'Minutes';
     }
 
     value = `${difference} ${timeUnit}`;
```

Comment:
i think it's here? or might be somewhere else. need a toFixed or a round.

<img width="420" alt="Screen Shot 2023-04-17 at 4 21 02 PM" src="https://user-images.githubusercontent.com/62641231/232613607-fb04e0f7-6886-4da2-9e55-3aaaaa7378e2.png">